scripts/eurostat/README.md

```diff
@@ -0,0 +1,45 @@
+# Importing EuroStat Regional Statistics by NUTS Into Data Commons
+
+
+> This README is a template of what an import's README might look like.
+
+Author: eftekhari-mhs
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+
+## About the Dataset
+
+### Download URL
+
+[TSV] file is available for download from <https://ec.europa.eu/eurostat/data/database> under 'Data navigation tree > Database by themes > General and regional statistics > Regional statistics by NUTS classification (reg)'.
+
+### Overview
+> The metadata for each dataset is available online at '<https://ec.europa.eu/eurostat/cache/metadata/en/<dataset name>_esms.htm>'

```

Comment:
How do we know which <datasetname> to sub in?

Reply:
it also doesn't render well in markdown

Reply:
I changed it to their general link.